eqcorrscan/utils/plotting.py

```diff
@@ -2184,6 +2184,122 @@ def subspace_fc_plot(detector, stachans, **kwargs):
     return fig
 
 
+@additional_docstring(plotting_kwargs=plotting_kwargs)
+def origintime_detections(detections, method='simple', **kwargs):
+
+    '''
+    Plot detections before and after applying match-filter and visual
+    comparison between them in one day.
+
+    :type detections: list
+    :param detections: list of eqcorrscan.core.match_filter.Detection
+    :type method: str
+    :param method: 'simple' or 'advance' defult to 'simple'
+    {plotting_kwargs}
+
+    :returns: :class:`matplotlib.figure.Figure`
+
+    .. warning::
+        For showing templates' origin, template name must be str of
+        the origin time of each template in match_filter or being changed to
+        origin time later.
+        Something like 2014-10-18T04:02:02
+        If template's name is something else, only detections will be plot.
+
+    .. rubric:: Example
+    >>> from eqcorrscan.core.match_filter import read_detections
+    >>> from eqcorrscan.utils.plotting import origintime_detections
+    >>> # Read text file detections
+    >>> detections = read_detections('path/to/file/filename')
```

Comment:
This fails because this file doesn't exist.

Reply:
> So is the purpose of this different to the cumulative detection plotting already included?
> I agree that plots are helpful, I just don't know where you would call this from, and what different/better information it gives than the cumulative plot, which is supported by `Party.plot()`?

You right. I'm not sure. Maybe this function is same as cumulative detection plot or even less usefull. Maybe it is another aspect of this mean.
If I'm being honest, in my case i had a simple mistake in my code then i detect duplicate of one event. after i saw this graph i found i had 2 events but only one point in figure.
![advance](https://user-images.githubusercontent.com/47061763/65393840-2a3bbd80-dd93-11e9-8343-d4c7d021b839.png)
![true](https://user-images.githubusercontent.com/47061763/65393844-39bb0680-dd93-11e9-8780-a00b2ccb959b.png)


I also noticed an issue in v0.3.3 that I don't know if there is in the new version or not.
I make a template of a specific day, then i use the template for this day to detect event.
but i did not see any detection or even self detection in cccs plot in match_filter function!
but after i draw my plot, i found there is self detection in detection and even more!
![true](https://user-images.githubusercontent.com/47061763/65393851-4d666d00-dd93-11e9-98dd-39a211c37198.png)
![cccsum_plot_2014-10-10T10_49_11_2014-10-10](https://user-images.githubusercontent.com/47061763/65393907-278d9800-dd94-11e9-94ce-0f5afe4a831b.png)